### ISSUE TITLE
Missing logo.txt file

### ISSUE THREAD

**Issue by cinnamon-bun** [cblgh/sector7557#1]
MacOS 10.13
node v12.16..1
npm 6.13.4

```
$ npm install --global sector7557

+ sector7557@0.0.6
```

```
$ sector7557 mud.cblgh.org

internal/fs/utils.js:230
    throw err;
    ^

Error: ENOENT: no such file or directory, open '/usr/local/lib/node_modules/sector7557/logo.txt'
    at Object.openSync (fs.js:457:3)
    at Object.readFileSync (fs.js:359:35)
    at Object.<anonymous> (/usr/local/lib/node_modules/sector7557/game.js:75:17)
    at Module._compile (internal/modules/cjs/loader.js:1158:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1178:10)
    at Module.load (internal/modules/cjs/loader.js:1002:32)
    at Function.Module._load (internal/modules/cjs/loader.js:901:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:74:12)
    at internal/main/run_main_module.js:18:47 {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: '/usr/local/lib/node_modules/sector7557/logo.txt'
}
```

```
$ ls /usr/local/lib/node_modules/sector7557/

LICENSE       constants.js  game.js*      package.json  types.js      walker.js
README.md     db.js         node_modules/ sectors.js    views/
```

**Comment by todrobbins** [comment]
macOS 10.15.6
node 12.18.0
npm 6.14.6

```js
➜ npm i -g sector7557
/Users/todrobbins/.nvm/versions/node/v12.18.0/bin/sector7557 -> /Users/todrobbins/.nvm/versions/node/v12.18.0/lib/node_modules/sector7557/game.js
+ sector7557@0.0.6
```


```js
➜ sector7557 mud.cblgh.org
internal/fs/utils.js:230
    throw err;
    ^

Error: ENOENT: no such file or directory, open '/Users/todrobbins/.nvm/versions/node/v12.18.0/lib/node_modules/sector7557/logo.txt'
    at Object.openSync (fs.js:458:3)
    at Object.readFileSync (fs.js:360:35)
    at Object.<anonymous> (/Users/todrobbins/.nvm/versions/node/v12.18.0/lib/node_modules/sector7557/game.js:75:17)
    at Module._compile (internal/modules/cjs/loader.js:1138:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1158:10)
    at Module.load (internal/modules/cjs/loader.js:986:32)
    at Function.Module._load (internal/modules/cjs/loader.js:879:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)
    at internal/main/run_main_module.js:17:47 {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: '/Users/todrobbins/.nvm/versions/node/v12.18.0/lib/node_modules/sector7557/logo.txt'
```

I'm using `nvm` so that may be a factor.

**Comment by cblgh** [comment]
fixed in `sector7557@0.0.7`, thanks!

(npm was adhering to my .gitignore, even tho i had force-pushed the txt-file to the repo)
